files/zh-cn/web/api/abortsignal/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: AbortSignal
+slug: Web/API/AbortSignal
+---
+{{APIRef("DOM")}}{{SeeCompatTable}}
+
+**`AbortSignal`** 接口表示一个信号对象（signal object），它允许您通过 {{domxref("AbortController")}} 对象与 DOM 请求（如 Fetch）进行通信并在需要时将其中止。
+
+## 属性
+
+_AbortSignal 接口还继承了其父接口 {{domxref("EventTarget")}} 的属性。_
+
+- {{domxref("AbortSignal.aborted")}} {{readonlyInline}}
+  - : 一个 {{domxref("Boolean")}}，表示*与之通信的请求是否被终止（true）或未终止（false）。*
+- {{domxref("AbortSignal.reason")}} {{readonlyInline}}
+  - : 一旦信号被中止，提供一个 JavaScript 值表示的中止原因。
+
+## 方法
+
+_**`AbortSignal`** 接口继续它父接口 {{domxref("EventTarget")}} 的方法。_
+
+- {{domxref("AbortSignal.throwIfAborted()")}}
+  - : 如果信号已经被中止，则抛出信号中止的 {{domxref("AbortSignal.reason", "reason")}}；否则没有任何用处。
+
+## 静态方法
+
+- {{domxref("AbortSignal.abort()")}}
+  - : 返回一个已经被设置为中止的 **`AbortSignal`** 实例。
+- {{domxref("AbortSignal.timeout()")}}
+  - : 返回一个在指定事件后将自动终止的 **`AbortSignal`** 实例。
+
+### 事件
+
+使用 [`addEventListener()`](/zh-CN/docs/Web/API/EventTarget/addEventListener) 或将事件监听器分配给该接口的 `oneventname` 属性。
+
+- [`abort`](/zh-CN/docs/Web/API/AbortSignal/abort_event)
+  - : 当与 signal 通信 DOM 请求被中止时调用。也可以通过 `onabort` 属性调用。
+- {{domxref("AbortSignal.onabort")}}
+  - : 当 {{event("abort_(dom_abort_api)", "abort")}} 事件触发时，即当信号正在与之通信的 DOM 请求被中止时调用。
+
+## 示例
+
+### 使用显示的信号终止 fetch 操作
+
+以下片段展示了我们如何使用 signal 去中止使用 [Fetch API](/zh-CN/docs/Web/API/Fetch_API) 下载视频。
+
+我们首先使用 {{domxref("AbortController.AbortController", "AbortController()")}} 创建一个中止控制器，然后使用 {{domxref("AbortController.signal")}} 属性获取与它联系的 {{domxref("AbortSignal")}} 对象的引用。
+
+当 [fetch 请求](/zh-CN/docs/Web/API/fetch) 开始时，我们将 `AbortSignal` 作为一个选项传递进请求的选项对象中（见下面的 `{signal}`）。这个将信号和控制器与 fetch 请求相联系，并且允许我们通过调用 {{domxref("AbortController.abort()")}} 中止它。你可以看见当中止按钮（`abortBtn`）被点击，第二个事件监听器触发，fetch 操作被中止。
+
+```js
+const controller = new AbortController();
+const signal = controller.signal;
+const url = 'video.mp4';
```

Comment:
please note the empty line in code sample, please copy it from github's editor (not preview)

```suggestion
const signal = controller.signal;

const url = 'video.mp4';
```